test/test_transforms_v2.py

```diff
@@ -1071,25 +1143,27 @@ def test_image_correctness(self, fn):
 
         torch.testing.assert_close(actual, expected)
 
-    def _reference_horizontal_flip_bounding_boxes(self, bounding_boxes):
+    def _reference_horizontal_flip_bounding_boxes(self, bounding_boxes, format):
```

Comment:
Do we need to pass the format as parameter here? I think we should be able to get it from the bounding_boxes parameter itself?